aries-site/src/data/structures/foundation.js

```diff
@@ -42,6 +42,29 @@ export const foundation = [
     logos in the experiences you create.`,
     sections: ['HPE Logo', 'HPE Element', 'Aruba Logo', 'Aruba Icon'],
   },
+  {
+    name: 'Developer Guidance',
+    category: 'Philosophy',
+    cardOrder: 2,
```

Comment:
Wondering if we can adjust the card order so this ends up on the first row so it's higher visibility. maybe instead of "human centered"?

Reply:
yeah I can do that I also wasn't sure where this should go